src/main/kotlin/sc/iview/SciView.kt

```diff
@@ -261,7 +261,10 @@ class SciView : SceneryBase, CalibratedRealInterval<CalibratedAxis> {
      * @param overwrite should the file be replaced, otherwise a unique incrementing counter will be appended
      */
     fun toggleRecordVideo(filename: String?, overwrite: Boolean) {
-        if (renderer is OpenGLRenderer) (renderer as OpenGLRenderer).recordMovie(filename!!, overwrite) else (renderer as VulkanRenderer).recordMovie(filename!!, overwrite)
+        if (renderer is OpenGLRenderer)
+                (renderer as OpenGLRenderer).recordMovie(filename!!, overwrite)
```

Comment:
Can we drop support for `OpenGLRenderer` now, since it is going away in scenery imminently?

Reply:
Oh hey, also, did you know that in Kotlin, you don't have to write this pattern? You can write:
```kotlin
if (renderer is OpenGLRenderer)
    renderer.recordMovie(filename!!, overwrite)
```
without the `(renderer as OpenGLRenderer)` cast? It's smart casting! :100:

Reply:
Ah nice!

Reply:
Actually there IntelliJ rejects it because renderer is mutable